lte/gateway/c/core/oai/tasks/amf/amf_app_state_converter.cpp

```diff
@@ -22,6 +22,9 @@ extern "C" {
 #include "lte/gateway/c/core/oai/common/log.h"
 }
 
+using magma::lte::oai::EmmSecurityContext;
+using magma::lte::oai::EmmSecurityContext_Count;
```

Comment:
this can be removed if using `auto` directive

Reply:
Comments Addressed